frontend/src/assets/scss/helpers/_mixins.scss

```diff
@@ -0,0 +1,8 @@
+/* ---------------------------------- *\
+            Mixins & helpers
+\* ---------------------------------- */
+
+/* responsive font-size */
+@function rem($size) {
+  @return calc($size / 16) * 1rem;
+}
```

Comment:
Why do we need this?
Tailwind already uses `rem`, so do we really need a converter?